webpack.config.js

```diff
@@ -72,7 +72,6 @@ module.exports = [
         },
         externals: {
             'decode-html': true,
-            'escape-html': true,
```

Comment:
Could you explain what's happening here?

Reply:
This line effectively tells Webpack to exclude `escape-html` from the output bundle based on the promise that it will be provided externally. Since it's now gone from our (non-dev) dependency list there's no need to exclude it from the output any more.